### PR TITLE
History

### PR DESCRIPTION
**Pull request by QuentinRG**

### CODE REVIEW

**Comment by yevhenverkhalantsev** [comment]
Rebase your branch from master first then I could to review your code

File: app/src/main/java/com/example/dashcarr/presentation/tabs/history/HistoryFragment.kt [hunk 1]
```diff
@@ -1,39 +1,139 @@
...
+/**
+ * HistoryFragment manages the display and interactions in the history tab.
+ */
 class HistoryFragment : BaseFragment<FragmentHistoryBinding>(
     FragmentHistoryBinding::inflate,
     showBottomNavBar = true
 ) {
     private val viewModel: HistoryViewModel by viewModels()
 
+    /**
```
Comment (yevhenverkhalantsev) [inline app/src/main/java/com/example/dashcarr/presentation/tabs/history/HistoryFragment.kt:29]: remove comments, todo is located under function

Comment (yevhenverkhalantsev) [reply app/src/main/java/com/example/dashcarr/presentation/tabs/history/HistoryFragment.kt:29]: and remove below the same comments

File: app/src/main/java/com/example/dashcarr/presentation/tabs/history/HistoryViewModel.kt [hunk 2]
```diff
@@ -1,20 +1,16 @@
...
-import com.example.dashcarr.data.entity.GeoPointEntity
 import com.example.dashcarr.domain.repository.IFirebaseDBRepository
 import dagger.hilt.android.lifecycle.HiltViewModel
-import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @HiltViewModel
 class HistoryViewModel @Inject constructor(
-    private val firebaseDBRepository: IFirebaseDBRepository
+    //private val firebaseDBRepository: IFirebaseDBRepository
```
Comment (yevhenverkhalantsev) [inline app/src/main/java/com/example/dashcarr/presentation/tabs/history/HistoryViewModel.kt:10]: remove. it was temporary val and fun

File: app/src/main/java/com/example/dashcarr/presentation/tabs/history/RecordingDetailsFragment.kt [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+) {
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        super.onCreateView(inflater, container, savedInstanceState)
+        return binding.root
+    }
+
+    /**
```
Comment (yevhenverkhalantsev) [inline app/src/main/java/com/example/dashcarr/presentation/tabs/history/RecordingDetailsFragment.kt:41]: remove comments

**Review by yevhenverkhalantsev — COMMENTED** [review]
All is good, we can improve it for sure in next tasks, I'm only missing design changes in new recordings.

**Comment by QuentinRG** [comment]
> All is good, we can improve it for sure in next tasks, I'm only missing design changes in new recordings.

**Review by yevhenverkhalantsev — APPROVED** [review]

**Pull request merged**